resources/testharness.js

```diff
@@ -2101,6 +2544,22 @@
         }
     };
 
+    /**
+     * Wrap a function so that it runs as a step of the current test.
+     *
+     * This allows creating a callback function that will run as a
+     * test step.
+     *
+     * @example
+     * let t = async_test("Example");
+     * onload = t.step_func(e => {assert_equals(e.name, "load"; t.done())})
```

Comment:
```suggestion
     * onload = t.step_func(e => {assert_equals(e.name, "load"); t.done()})
```

But I wonder if omitting `t.done()` wouldn't make for a better example, since otherwise this could use `t.step_func_done`...

Reply:
I think it's hard to think of an example that's, small, complete, and doesn't benefit from `step_func_done`. Given that I'd rather do something slightly non-idomatic, but correct, than have an example that's either complex or incomplete.